lib/react/jsx/processor.rb

```diff
@@ -0,0 +1,17 @@
+module React
+  module JSX
+    class Processor
+      def self.instance
+        @instance ||= new
+      end
+      
+      def self.call(input)
+        instance.call(input)
```

Comment:
Do we benefit from having `@instance` ? What if this class method was simply 

```ruby
def self.call(input)
  JSX::transform(input[:data])
end
```

would that be sufficient?